calculator.js

```diff
@@ -1,2 +1,74 @@
+var value = null
```

Comment:
This is ok to use an outside variable, but we really wanted the `value` to be a property of your `calculator` object:

```js
var calculator = {
  value: 0,
  add (a, b) {
//...
}
```

And then reference it from your methods using `this.value`